protostar/cli/network_command_mixin.py

```diff
@@ -60,9 +60,9 @@ def get_network_config(args) -> NetworkConfig:
         )
 
     @staticmethod
-    def get_gateway_client(args) -> GatewayFacade:
+    def get_gateway_client(args) -> GatewayClient:
         network_config = NetworkCommandMixin.get_network_config(args)
         return GatewayClient(
             net=network_config.gateway_url,
-            chain=network_config.chain_id,
+            chain=network_config.chain_id,  # type: ignore
```

Comment:
```suggestion
            chain=network_config.chain_id,
```

why?

Reply:
i should explain in the comment, right. This one is expecting chain_id of `StarknetChainId` type, although starknet.py will accept any value (numeric included) and will work just fine

Reply:
In fact this is not even used in the client. The constructor is going to get updated pretty soon (0.10)

Reply:
OK.